src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    autocomplete="off"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:
+          <input class="input" type="text"
+          name="Surname" id="surname"
+          required
+            >
```

Comment:
```suggestion
          <input 
            class="input"
            type="text"
            name="Surname"
            id="surname"
            required
          >
```
Форматируй теги в которых много атрибутов вот так